pkg/sdk/logging/model/output/opensearch_test.go

```diff
@@ -50,6 +50,7 @@ buffer:
 	reload_connections true
 	scheme https
 	ssl_verify false
+  ssl_version TLSv1_2
```

Comment:
nit: not a new thing, but would you please replace tabs with spaces (or vice versa) so that the indentation stays consistent?